### PR TITLE
Implemented the svdvals and test

### PR DESCRIPTION
**Pull request by SIMRA99**

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 1]
```diff
@@ -368,3 +368,35 @@ def test_torch_svd(
         atol=1e-2,
         ground_truth_backend=frontend,
     )
+    
+    
+#svdvals
+@handle_frontend_test(
+    fn_tree="torch.linalg.svdvals",
+    dtype_and_x=_get_dtype_and_square_matrix(),
+    n=helpers.ints(min_value=2, max_value=5),
```
Comment (Aarsh2001) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:377]: You've sampled an argument `n`, but haven't passed it in the test function. Also as far as this function is concerned, there doesn't exist any `n` keyword argument in it's implementation. Kindly remove it, thanks !

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py (hunk 1 above)
Comment (SIMRA99) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py]: I have removed the argument. Please review. Thanks !

**Review by Aarsh2001 — APPROVED** [review]
LGTM !

**Comment by SIMRA99** [comment]
#7667

**Comment by SIMRA99** [comment]
Close #7667

**Pull request merged**